cart/models.py

```diff
@@ -1,52 +1,86 @@
 from django.db import models
-
 from accounts.models import Profile
 from products.models import Product
 
 
-
-class Order(models.Model):
-    user_profile = models.ForeignKey(Profile, on_delete=models.SET_NULL, blank=True, null=True)
-    date_ordered = models.DateTimeField(auto_now_add=True)
-    complete = models.BooleanField(default=False, null=True, blank=False)
-    transaction_id = models.CharField(max_length=200, null=True)
+class Cart(models.Model):
+    user_profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
     
     @property 
     def get_cart_total(self):
-        orderitems = self.orderitem_set.all()
```

Comment:
this is really bad:
1. get_* method has delete inside it - don't do it.
2. Using all .all() and then iterating in Python is anti pattern use [aggregation](http://docs.djangoproject.com/en/4.2/topics/db/aggregation/#topics-db-aggregation)